src/cargo/ops/cargo_rustc/fingerprint.rs

```diff
@@ -421,6 +430,7 @@ pub fn prepare_build_cmd<'a, 'cfg>(cx: &mut Context<'a, 'cfg>, unit: &Unit<'a>)
         deps: Vec::new(),
         local: local,
         memoized_hash: Mutex::new(None),
+        rustflags: cx.rustflags_args(unit),
```

Comment:
This I think can just be `Vec::new()` because the build script *executing* won't depend on RUSTFLAGS, just the compilation (handled above)